pydocx/tests/test_docx.py

```diff
@@ -123,6 +123,13 @@ def test_result_from_file_pointer_matches_result_from_path(self):
         assert file_html
         self.assertEqual(path_html, file_html)
 
+    def test_googledocx_parsing(self):
+        try:
+            Docx2Html(path=self.get_path_to_fixture('export_from_googledocs.docx')).parsed
+        except ValueError:
+            self.fail("Can't parse google docs exported docx")
```

Comment:
Instead of adding this separate test case, you can add `'export_from_googledocs'` to the list here: https://github.com/CenterForOpenScience/pydocx/blob/master/pydocx/tests/test_docx.py#L36. Test cases are automatically created from that list of input/output files.